Gui/opensim/view/src/org/opensim/view/motions/MotionDisplayer.java

```diff
@@ -961,7 +960,7 @@ public void updateMotionObjects(){
         if (simmMotionData instanceof AnnotatedMotion){
             // Add place hoders for markers
             AnnotatedMotion mot= (AnnotatedMotion) simmMotionData;
-            setExperimentalMarkerScaleFactor(mot.getDisplayForceScale());
```

Comment:
Was this a separate bug that you caught? If not, could you explain the reason for this change?

Reply:
Yes, it was a separate bug that I caught.